### ISSUE TITLE
Create Docker Container for Verification

### ISSUE THREAD

**Issue by shahzaibk23** [merledu/nucleusrv#72]
To Create a Docker Image and Container for Verification.

Image should have:
- RISC-V Sail ISS
- Spike
- RV GCC
- RISCOF

**Comment by latifbhatti** [comment]
I have completed docker image. firstly install docker by this command 
```sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin```

first time install riscv gcc tool chain for vector extension, second command is docker pull 
```latif040/rvv_gcc_tool_chain:1.0```
 then check docker image by this command 
```docker images```
and finally run this command 
```sudo docker run --rm -it latif040/rvv_gcc_tool_chain:1.0```

now give the assembly file to docker image. firstly save the file in same directory(home/app) touch file name.s open the file in editor vim f1.s then to generate object file 
```riscv32-unknown-elf-gcc -c -march=rv32imv -o f1.o f1.s and finally to get machine code riscv32-unknown-elf-objdump -d test.o | awk '{print $2}' | grep -E '^[0-9a-f]+$'```

**Comment by hadirkhan10** [comment]
@shahzaibk23 why is this issue still open? Any pending changes left?

**Comment by shahzaibk23** [comment]
Yes, The Docker image that @latifbhatti has created contains only RISC-V GCC and Spike ISS.

RISC-V SAIL ISS and RISCOF Framework are still under development by @SyedHassanUlHaq
